docs/zh/v0.4/developer-guide/datanode/query-engine.md

```diff
@@ -43,14 +29,15 @@ data columns, the mainly used persistent file format [Apache Parquet][3] in Grep
 provides multi-level indices and filters that make it easy to prune data during querying. In the future, we
 will make more use of this feature, and develop our separated index to handle more complex use cases.
```

Comment:
👆The English content can be removed now.

Reply:
@waynexia LGTM, and please take a look at the comment from @nicecui.